locales/en.json

```diff
@@ -17,28 +17,19 @@
   },
   "items": {
     "battery_storage_installation": "Battery Storage Installation",
-    "ebike": "E-Bike",
```

Comment:
Did we intend to drop so many of these? I couldn't see how these modifications were related to the vo_item changes so I just wanted to raise.

Reply:
This is because I set the `items` and `urls` fields of the locale files to contain IRA items only. (The strings are only used in v0, and v0 only contains IRA incentives, and those don't use any of the removed items.)